lib/posexional/file.ex

```diff
@@ -76,6 +76,34 @@ defmodule Posexional.File do
     end)
   end
 
+  @spec stream(Enumerable.t(), %Posexional.File{}) :: Enumerable.t()
+  def stream(str, _file = %{separator: separator, rows: rows}) do
+    str
+    |> Stream.concat([separator])
```

Comment:
why do we need the separator at the end?

Reply:
It is required since Stream.transform unfortunately does not return the latest accumulator. So, using string.split, we would lose the last file entry if it does not have a final separator. I have added this case as test

I've tried a bit to get rid of it, but i failed and havent find anything useful in the docs =/